alphazero/_game.py

```diff
@@ -7,149 +7,149 @@
 class Game:
     def __init__(self, *args):
         """
-        Construct an instance of the game class.
-        """
+		Construct an instance of the game class.
```

Comment:
Can we keep it to spaces please (opening up a can of worms here haha)

Reply:
Yeah I'll change it, not sure if black changed all of that stuff or something because I didnt touch those lines lol

Reply:
Just committed hopefully it will look better now.

Reply:
Yup, diff looks much cleaner. Thanks!